### PR TITLE
feat(@kampus-apps/pano): add discord strategy for remix-auth

### PR DESCRIPTION
**Pull request by andyanday33**
Solves #242

### CODE REVIEW

File: apps/pano/package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "react-copy-to-clipboard": "5.1.0",
     "react-dom": "18.2.0",
     "remix-auth": "3.4.0",
+    "remix-auth-discord": "^1.2.1",
```
Comment (usirin) [inline apps/pano/package.json:52]: let's remove the carrot, the easiest way to do this:

```sh
cd apps/pano
npm uninstall remix-auth-discord && npm install remix-auth-discord --save-exact
```

File: apps/pano/app/routes/api/auth/discord.callback.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// app/routes/auth/discord.callback.tsx
+import type { LoaderFunction } from "@remix-run/node";
+import { authenticator } from "~/authenticator.server";
+
+export let loader: LoaderFunction = ({ request }) => {
+  return authenticator.authenticate("discord", request, {
```
Comment (cansirin) [inline apps/pano/app/routes/api/auth/discord.callback.tsx:6]: i think we are missing `await` here

File: apps/pano/app/features/authenticator/strategies.ts [hunk 3]
```diff
@@ -95,4 +97,28 @@ export const strategies = {
...
+        },
+      });
+
+      if (!user) {
+        user = await prisma.user.create({
+          data: { email: profile.__json.email!, username: profile.displayName },
+        });
+      }
+
+      if (!user) throw new AuthorizationError("kullanici bulunamadi");
```
Comment (cansirin) [inline apps/pano/app/features/authenticator/strategies.ts:120]: can you please write the error message with Turkish characters?

File: apps/pano/app/routes/api/auth/discord.callback.tsx (hunk 2 above)
Comment (andyanday33) [reply apps/pano/app/routes/api/auth/discord.callback.tsx:6]: Neither the example for auth strategies nor @usirin 's code includes await for this component, should I add it anyways?

**Review by usirin — APPROVED** [review]
💯

**Pull request merged**